lambdas/farsetlabs/handlers/schemas/event-schema.json

```diff
@@ -110,7 +110,6 @@
       "required": [
         "name",
         "address",
-        "city",
```

Comment:
No longer considering city a required field for a venue (what if it's not in a city?)

Reply:
`city` as an optional field makes sense; we can always decorate this later from the address.